System.IO.Abstractions/FileInfoBase.cs

```diff
@@ -6,7 +6,7 @@ namespace System.IO.Abstractions
     [Serializable]
     public abstract class FileInfoBase : FileSystemInfoBase
     {
-        protected FileInfoBase(IFileSystem fileSystem) : base(fileSystem)
+        protected FileInfoBase(IFileSystem fileSystem, string path) : base(fileSystem, path)
```

Comment:
To reduce public API changes and therefore downstream churn please undo the changes in this file and override `ToString` in `*Wrapper`/`Mock*` instead.

Reply:
ToString moved to MockFileInfo